quark/script/__init__.py

```diff
@@ -579,3 +628,47 @@ def _wrapMethodObject(
             for caller in list(caller_set)
         ]
     return caller_methods
+
+
+def checkMethodCalls(
+        samplePath: PathLike,
+        targetMethod: Union[Tuple[str, str, str], MethodObject],
+        checkMethods: List[Tuple[str, str, str]]) -> bool:
+    """Check if any of the specific methods shown in the `targetMethod`
+
+    :param samplePath: target file
+    :param targetMethod: python list contains the class name,
+                         method name, and descriptor of the target method
+                         or a Method Object.
+    :param checkMethods: python list contains the class name,
+                         method name, and descriptor of the target method
+
+    :return: bool that indicate specific methods can be called or defined within a `target method` or not.
+    """
+    _target_methods = []
```

Comment:
Since we are using lower camel case naming in Quark script, please change the variables naming type.